pkg/standalone/standalone.go

```diff
@@ -146,9 +146,11 @@ func Init(runtimeVersion, dashboardVersion string, dockerNetwork string, slimMod
 		if !dockerInstalled {
 			return errors.New("could not connect to Docker. Docker may not be installed or running")
 		}
-		defaultImageRegistryName, err = utils.GetDefaultRegistry(githubContainerRegistryName, dockerContainerRegistryName)
-		if err != nil {
-			return err
+		if useDefaultReg(imageRegistryURL, fromDir) {
```

Comment:
If this is only used here, we can also simply add the condition here instead of creating a one-liner function.